docs/README.md

```diff
@@ -1,8 +1,85 @@
-Documentation
-====
+# Website
 
-All documentation for Arcus Background Jobs which are published via GitHub Pages with Jekyll.
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
 
-## Running Jekyll locally
+## Installation
 
-Running Jekyll locally to test your changes is super easy, learn more about it [here](https://jekyllrb.com/docs/#instructions).
\ No newline at end of file
+```console
+yarn install
```

Comment:
Maybe a good thing to include the path where these commands should be executed. As we are somewhat newbies at this 😄 .